Makefile

```diff
@@ -51,28 +45,30 @@ GOMEGA_VERSION := $(shell awk '/github.com\/onsi\/gomega/ {print $$2}' go.mod)
 # Test coverage threshold
 export COVERAGE_MIN ?= 75
 
-LOCAL_OS := $(shell uname)
-ifeq ($(LOCAL_OS),Linux)
-    TARGET_OS ?= linux
-    XARGS_FLAGS="-r"
-else ifeq ($(LOCAL_OS),Darwin)
-    TARGET_OS ?= darwin
-    XARGS_FLAGS=
-else
-    $(error "This system's OS $(LOCAL_OS) isn't recognized/supported")
-endif
+# Image URL to use all building/pushing image targets;
+# Use your own docker registry and image name for dev/test by overridding the IMG and REGISTRY environment variable.
+IMG ?= $(shell cat COMPONENT_NAME 2> /dev/null)
+REGISTRY ?= quay.io/stolostron
+TAG ?= latest
 
-.PHONY: fmt lint test coverage build build-images
+# go-get-tool will 'go install' any package $2 and install it to $1.
+define go-get-tool
+@set -e ;\
+echo "Checking installation of $(2)" ;\
+GOBIN=$(PWD)/bin go install $(2)
```

Comment:
Should this be `GOBIN=$(LOCAL_BIN)`?

Actually, this seems a little confusing along with the other `GOPATH` and `GOBIN` definitions above... I'm not sure what we want to happen: should things be installed to `./.go/bin/`, or just `./bin/`?

Reply:
I think it makes sense :D